tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -76,6 +76,13 @@ def test_invalid_characters(self):
         assert_equal(return_code, 1)
         assert_equal(len(invalid_specs), 2)
 
+    def test_unknown_exception(self):
+        args = pat.setup_parser().parse_args('test --path tests/fixtures --filter RuleID=Example.Rule.Unknown.Exception'.split())
+        args.filter = pat.parse_filter(args.filter)
+        return_code, invalid_specs = pat.test_analysis(args)
+        assert_equal(return_code, 1)
+        assert_equal(len(invalid_specs), 2)
+
```

Comment:
I couldn't recreate the original problem listed in the issue. With `test`, `zip`, and `upload`, unexpected exceptions are added to the failed summary, so I simply added this new test.

```
bin/panther_analysis_tool test --path ./tests/fixtures/valid_analysis --filter RuleID=AWS.CloudTrail.MFAEnabled
[INFO]: Testing analysis packs in ./tests/fixtures/valid_analysis

AWS.CloudTrail.MFAEnabled
        [PASS] Root MFA not enabled fails compliance
        [PASS] User MFA not enabled fails compliance
[WARNING]: Unexpected exception: {}

--------------------------
Panther CLI Test Summary
        Path: ./tests/fixtures/valid_analysis
        Passed: 0
        Failed: 1
        Invalid: 0

--------------------------
Failed Tests Summary
        AWS.CloudTrail.MFAEnabled
                ['User MFA enabled passes compliance.']
```